sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -406,6 +406,31 @@ ze_result_t ZeCall::doCall(ze_result_t ZeResult, const char *CallStr,
     return mapError(Result);
 #define ZE_CALL_NOCHECK(Call) ZeCall().doCall(Call, #Call, false)
 
+// This helper function creates a pi_event and associate a pi_queue.
+// We need to increment the reference counter here to avoid pi_queue
+// being release before the associated pi_event is released because
+// piEventRelease requires access to the associated pi_queue.
+// In piEventRelease, the reference counter is decremented to release it.
+inline pi_result createEventAndAssociateQueue(
+    pi_queue Queue, pi_event *Event, pi_command_type CommandType,
+    ze_command_list_handle_t ZeCommandList, ze_event_handle_t *ZeEvent,
+    bool QueueLockAcquired) {
```

Comment:
Please remove the last argument, and describe the others.

Reply:
Done. Thanks.